plugin/evm/atomic/sync/atomic_sync_extender.go

```diff
@@ -9,22 +9,43 @@ import (
 	"github.com/ava-labs/avalanchego/database/versiondb"
 	syncclient "github.com/ava-labs/coreth/sync/client"
 
-	"github.com/ava-labs/coreth/plugin/evm/atomic/state/interfaces"
 	"github.com/ava-labs/coreth/plugin/evm/message"
 	"github.com/ava-labs/coreth/plugin/evm/sync"
+	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/log"
 )
 
 var _ sync.Extender = (*AtomicSyncExtender)(nil)
 
+type AtomicBackend interface {
```

Comment:
IMO this can also be a concrete type, as it seems acceptable to me for sync to import state package.